test/test_torrent.cpp

```diff
@@ -263,7 +278,6 @@ TORRENT_TEST(torrent)
 		std::vector<char> piece(128 * 1024);
 		for (int i = 0; i < int(piece.size()); ++i)
 			piece[i] = (i % 26) + 'A';
-
```

Comment:
please add back this blank line too. these tests have changed in master (I'm pretty sure) and this seems like an unnecessary conflict.

Reply:
This is commented code, that was removed in original PR. Are you sure it really make sense?

Reply:
it will still generate a conflict with master.